src/components/camera-tool.js

```diff
@@ -275,7 +277,7 @@ AFRAME.registerComponent("camera-tool", {
 
     const label = this.data.label;
     const isFrozen = this.el.sceneEl.is("frozen");
-    const hasDuration = this.data.captureDuration !== 0 && this.data.captureDuration !== Infinity;
+    const hasDuration = this.data.captureDuration !== Infinity;
     const isPhoto = this.data.captureDuration === 0;
```

Comment:
Looks like `this.data.captureDuration` won't be `0` anymore so this `isPhoto` check is suspicious.